src/main/java/com/beantastic/api/models/dto/EnemyDTO.java

```diff
@@ -0,0 +1,74 @@
+package com.beantastic.api.models.dto;
+
+import java.util.*;
+
+import com.beantastic.api.models.entities.BeanClassStatistic;
+import com.beantastic.api.models.entities.Enemy;
+import com.beantastic.api.models.entities.EnemyAction;
+import com.beantastic.api.models.entities.EnemyDifficultyStatistic;
+import com.beantastic.api.models.entities.EnemyStatisticModifier;
+import com.beantastic.api.models.entities.StatisticType;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class EnemyDTO {
+
+    @JsonProperty("name")
```

Comment:
Just put them there for now, I also do not think they are needed but atm since I was creating fake data it didnt seem to like making it JSON without them. I think if I add the @Data tag instead it may work

Reply:
Damn, I can update your comments D: I think you tagged an actual user so I just removed the @. I did not think that would work